portfolio.html

```diff
@@ -44,7 +44,30 @@ <h2>projects by the databrew team</h2>
                     </section>    
                     
                 <!-- Portfolio -->
-         <section id="portfolioifc">
+                <section id="portfoliobohemia">
+					<header>
+						<h6><strong>Data Collection, Management, and Analysis for BOHEMIA</strong></h6><br>
+                        <p>BOHEMIA (Broad One Health Endectocide-based Malaria Intervention in Africa) is a 3-year Unitaid-funded project that will evaluate the mass drug administration of the antiparasitic drug ivermectin to humans and/or livestock in Tanzania and Mozambique during the malaria season. Databrew will assist in collecting and analyzing data on the epidemiological impact of this large-scale research project, which will ultimately support normative guidance, global and national policy change.</p>
+                    </header>
+				</section> 
+               
+                <section class="wrapper style1 container special">
+                       <div class="row" align="left">
+                           <div class="6u 12u(narrower)">
+                           <a class="image featured"><img src="images/law_as_data.png" style="max-width: 500px;"/></a>
+                           </div>
+                            <div class="6u 12u(narrower)">
+                                <header>
+                                <h5><strong>Data Visualizations for Santa Fe Institute's book <i>Law As Data</i></strong></h5>
+                                <p>Databrew analyzed data and created the visualizations (bar charts, violin plots, scatterplots) for <a href="https://www.santafe.edu/news-center/news/new-book-law-data-explores-burgeoning-field-computational-legal-analysis" target="_blank"><i>Law as Data</i></a>, a book published by the Santa Fe Institute which explores the new field of computational legal analysis — the study of the law that uses legal texts as data.</p>
```

Comment:
I suggest we remove the text in parentheses ("(bar cahrts, violin plots, scatterplots)").